src/Microsoft.Extensions.Logging.Testing/LoggedTest/ILoggedTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using Xunit.Abstractions;
+
+namespace Microsoft.Extensions.Logging.Testing
+{
+    internal interface ILoggedTest
```

Comment:
All properties `{ get; }` only.

Reply:
TestOutputHelper is set via the Invoker so we need to at least maintain the setter on that one. I could remove the other ones but I guess we can just drop the interface completely.

Reply:
The interface offers an escape hatch if we want to change the logging framework behaviour without extending from the base class. If we had this, we would have been able to implement a SinklessLoggedClass in Kestrel instead of having to make a cross repo change here.